FirebaseStorage/Tests/Integration/FIRStorageIntegrationTests.m

```diff
@@ -510,10 +511,11 @@ - (void)testGetFile {
                       NSString *data = [NSString stringWithContentsOfURL:fileURL
                                                                 encoding:NSUTF8StringEncoding
                                                                    error:NULL];
-                      NSString *expectedData = @"Hello World";
-                      XCTAssertEqualObjects(data, expectedData);
-                      XCTAssertEqualObjects([snapshot description], @"<State: Success>");
-                      [expectation fulfill];
+                      [ref dataWithMaxSize:128
```

Comment:
nit: maybe use `fileData.size` so this test doesn't break if fileData changes

Reply:
Fixed